### DISCUSSION TITLE
SQL Query Logging

### DISCUSSION THREAD

**Discussion by vajlo** [serenity-is/Serenity#5546] [Help]
Hello,

is there possibility to enable loging sql queries in .Net core 5 Serenity as it was in ASP.NET MVC to file or to console while debugging?  Or is there any way to do something like that?

Thanks & Regards,
vajlo

**Comment by dfaruque** [comment]
yes, I am also looking for "How to enable logging sql queries in .NET 5 Serenity?"
anyone knows how?

**Reply by osdan-madrid** [reply]
Did you find it?
